cfg/cfg.go

```diff
@@ -121,6 +125,10 @@ func setLogLevel() {
 func Parse() {
 	flag.Parse()
 
+	if PodSelector == "" {
```

Comment:
That's not possible in go to evaluate if string flag is explicitly set "" or it's missing.
And technically "" pod selector is a correct one and will match all the pods but che-machine-exec is not really adapted to work with multiple pods, so I choose failing if it's "".

The previous solution with failing if pod selector is not found, seems to have a bug, and it's going to fail if CHE_WORKSPACE_ID is missing even if custom pod selector is configured with argument (POD_SELECTOR env var should work fine)